tick/survival/simu_coxreg.py

```diff
@@ -202,3 +215,274 @@ def _as_dict(self):
         dd.pop("times", None)
         dd.pop("censoring", None)
         return dd
+
+
+class SimuCoxRegWithCutPoints(SimuWithFeatures):
+    """Simulation of a Cox regression for proportional hazards with cut-points
+    effects in the features
+
+    Parameters
+    ----------
+    features : `numpy.ndarray`, shape=(n_samples, n_features), default=`None`
```

Comment:
There is something that I don't understand. We don't pass coefficients to this simulation class ?

Reply:
Indeed: the purpose of the class is to simulate Cox model regression data with cut-points effects in the features. To make sure we get `real` cut-points effects, the later are generated as features quantiles (and hence neither passed to the class), then the coefficients we called `coeffs_binarized` are generated according to the cut-points. In general, the number of coefficients per raw feature are not known in advance. We could also change the class so that we pass the cut-points as well as the binarized coefficients, but with the risk of getting invisible effects (and not answering the aim of the class).